File: ass13/program1.c
/*Accept number from user and display below pattern. 
Input :  5  
Output : A B C D E
*/
#include<stdio.h>

void Pattern(int iNo)
{
    int iCnt = 0;
    
    if(iNo < 0)         // Updator
    {
        return;
    }
    for(iCnt = 1;iCnt <= iNo;iCnt++)
    {
         switch (iCnt)
         {
         case 1:
            printf("A");
             break;
             case 2:
            printf("B");
             break;
             case 3:
            printf("C");
             break;
             case 4:
            printf("D");
             break;
             case 5:
            printf("E");
             break;
         
         default:
            printf("nothing");
             break;
         }
    }
}   

int main()
{
    int iValue = 0;
    
    printf("Enter number\n");
    scanf("%d",&iValue);
    
    Pattern(iValue);
    
    return 0;
}

File: ass66/Program1.java
/*
1. Write Java program which accept two arrays from user and display contents of each array. 
Input :  2 9 7 5 2 3   9 3 5 5 
Output :  2 9 7 5 2 3   9 3 5 5
*/
import java.lang.*;
import java.util.*;
class MyArray
{
    void Display(int arr[],int brr[])
    {
        for( int i = 0; i<arr.length; i++)
		{
			System.out.print(arr[i]+"");
        }
        System.out.println("");
        for( int j = 0; j<brr.length; j++)
		{
			System.out.print(brr[j]+"");
        } 

    }
}

class Program1
{
	public static void main(String arg[])
	{
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter number of first array");
		int no1 = sobj.nextInt();

		System.out.println("Enter number of second array");
        int no2 = sobj.nextInt();

        int arr[] = new int[no1];
        int brr[] = new int[no2];

		System.out.println("Enter the first array");

		for( int i = 0; i<arr.length; i++)
		{
			arr[i]= sobj.nextInt();
        }
        System.out.println("Enter the second array");
        for( int j = 0; j<brr.length; j++)
		{
			brr[j]= sobj.nextInt();
        }
        MyArray dobj = new MyArray();
        dobj.Display(arr,brr);

        
	}
}